src/controllers/tasks/register.locations.controller.ts

```diff
@@ -40,3 +49,21 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
     return next(e);
   }
 };
+
+const formatSailForDisplay = (regLoc: RegisterLocation[]): string => {
+  let sailAddress = "";
+  if (regLoc.length && regLoc[0].sailAddress) {
+    sailAddress = formatAddressForDisplay(formatAddress(regLoc[0].sailAddress));
+  }
+  return sailAddress;
+};
+
+const formatRegisterForDisplay = (regLoc: RegisterLocation[]): Array<string> => {
+  const registers = new Array<string>();
+  for (const key of Object.values(regLoc)) {
```

Comment:
i'd rename the key var to value as you are iterating through the object.values here